improver/synthetic_data/set_up_test_cubes.py

```diff
@@ -539,6 +542,7 @@ def set_up_variable_cube(
     realizations: Optional[Union[List[float], ndarray]] = None,
     vertical_levels: Optional[Union[List[float], ndarray]] = None,
     pressure: bool = False,
+    height: bool = True,
```

Comment:
As above, please add height to the docstring below the pressure flag. And consider changing height default to False.